### PR TITLE
Update to tabula/adding data to mongo

### PR DESCRIPTION
**Pull request by ValeryiaPilchuk**
updated extracting the information needed from our pdf (syllabus), added info page (where you are able to upload pdf) that was pushed but never merged, began adding data to mongo (still needs work mostly cause im having some unrelated bugs)

### CODE REVIEW

File: client/src/components/pdf_scan.py [hunk 1]
```diff
@@ -1,31 +1,75 @@
...
+
+
+    with open(csv_assignment, 'r') as read_obj:
+
+        # pass the file object to DictReader() to get the DictReader object
+        csv_reader = reader(read_obj)
+        csv_dict_reader = DictReader(read_obj)
+
+        # get column names from a csv file
+        column_names = csv_dict_reader.fieldnames
```
Comment (yywu123) [inline client/src/components/pdf_scan.py:43]: Any purpose use global in here ?

**Review by yywu123 — APPROVED** [review]
Great documentation on what you are doing. It make it clear for me to understand stuff.

File: models/NeverLateModel.js [hunk 2]
```diff
@@ -29,9 +29,13 @@ const neverLateSchema = new mongoose.Schema(
                     type: String,
                     require: true
                 },
+                Description: {
```
Comment (tenzin1308) [inline models/NeverLateModel.js:32]: what is desc for in the database?

File: models/NeverLateModel.js [hunk 3]
```diff
@@ -29,9 +29,13 @@ const neverLateSchema = new mongoose.Schema(
...
+                    type: String,
+                    require: false
+                },
                 StartTime: {
                     type: String,
-                    require: true
```
Comment (tenzin1308) [inline models/NeverLateModel.js:34]: why was start time changed to false?? we need it so we can display on the calendar.
please change it back to {require: true}

**Review by tenzin1308 — COMMENTED** [review]
Not sure how you are connecting your Input.js (frontend) with backend.
Also if you could put those python code in folder called "python"  python_tests.py or pdf_scan.py

**Comment by bhusalanil121** [comment]
Nice idea to deal with Pdf (Update + scan) .... I like the way you attached testing at the same time for the pull request.

**Comment by bhusalanil121** [comment]
@ValeryiaPilchuk, If there is nothing to change for this pull request we can merge it main. Looks good to me.

**Pull request merged**